docker-compose.yml

```diff
@@ -0,0 +1,20 @@
+version: '3'
+
+services:
+  web:
+    container_name: apiserver
+    restart: always
+    build: .
+    ports:
+      - '5000:3000'
+    links:
+      - mongo
+    volumes:
+      - .:/app
+  mongo:
+    container_name: mongodatabse
+    image: mongo
+    ports:
+      - '27018:27018'
```

Comment:
This won't work unless you configure your mongo server port to `27018`. If you just want to export `27018` because you might have another mongo container using this port, just map `27018:27017`